core/node/block_reverter/src/lib.rs

```diff
@@ -419,18 +421,46 @@ impl BlockReverter {
                 });
             combine_results(&mut overall_result, result);
 
-            for chunk_id in 0..snapshot.storage_logs_filepaths.len() as u64 {
+            let mut is_incomplete_snapshot = false;
+            let chunk_ids_iter = (0_u64..)
+                .zip(&snapshot.storage_logs_filepaths)
+                .filter_map(|(chunk_id, path)| {
+                    if path.is_none() {
+                        if !is_incomplete_snapshot {
+                            is_incomplete_snapshot = true;
+                            tracing::warn!(
+                                "Snapshot for L1 batch #{} is incomplete (misses al least storage logs chunk ID {chunk_id}). \
+                                 It is probable that it's currently being created, in which case you'll need to clean up produced files \
+                                 manually afterwards",
+                                snapshot.l1_batch_number
+                            );
+                        }
+                        return None;
```

Comment:
Maybe, it does make sense to attempt removing "future" storage log chunks?

Reply:
I think it doesn't - snapshot creator shouldn't be running while a rollback is taking place. Let's keep as is and remember/revisit when we revamping the block revertion next wave